tools/deploy.sh

```diff
@@ -36,7 +36,7 @@ fi
 if [ "${DEPLOY_TLS}" == "true" ]; then
     BMO_SCENARIO="${BMO_SCENARIO}/tls"
     IRONIC_SCENARIO="${IRONIC_SCENARIO}/tls"
-else
+elif [ "${DEPLOY_BASIC_AUTH}" == "true" ]; then
```

Comment:
So BMO_SCENARIO will be unset if we're using no authorization. Is that OK? Maybe we don't use/support that mode any more?

Reply:
Good point. May be we should be more explicit here. What's your suggestion

Reply:
For now I would be happy with an else statement that just exited with an error. We can add noauth support back if it actually comes up as something we need.

Reply:
What do you think of something like https://github.com/metal3-io/baremetal-operator/pull/681?

Reply:
Fair enough. I ll put a patch